### PR TITLE
Migrated test_updates_in_options_file_on_quorum_changes.py

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py)

Made modifications as per the redant pattern and ops.

Updates: #292

Fixes: #363

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

File: tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        ret = redant.execute_abstract_op_node(cmd, self.server_list[0])
+        out = ret['msg'][0].rstrip('\n')
+        new_global_op_ver = out.split('=')[1]
+
+        if int(prev_global_op_ver) + 1 != int(new_global_op_ver):
+            raise Exception("Failed:The global-option-version didn't change"
+                            " on a volume set operation")
+
+        redant.logger.info("The global-option-version was successfully "
+                           " updated in the options file")
```
Comment (nishith-vihar) [inline tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py:63]: I think success logging should not be done here.

File: tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+        # Setting Quorum ratio in percentage
+        quorum_perecent = {'cluster.server-quorum-ratio': '70%'}
+        redant.set_volume_options('all', quorum_perecent, self.server_list[0])
+
+        # Checking 'options' file for quorum related entries
+        cmd = "cat /var/lib/glusterd/options | grep global-option-version"
+        ret = redant.execute_abstract_op_node(cmd, self.server_list[0])
+        out = ret['msg'][0].rstrip('\n')
+        new_global_op_ver = out.split('=')[1]
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py:56]: I think this repetitive code can be converted into a function.

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py:63]: Screenshot from 2021-05-19 10-33-21
I think we need this logging here :)

File: tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py (hunk 2 above)
Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_updates_in_options_file_on_quorum_changes.py:56]: Ack

**Review by nik-redhat — APPROVED** [review]

**Comment by nik-redhat** [comment]
SS of a successful run, after this commit?

**Comment by aujjwal-redhat** [comment]
Screenshot from 2021-05-19 13-02-41

**Pull request merged**